k8s/custom-governance/README.md

```diff
@@ -168,12 +169,10 @@ gcloud auth configure-docker
   * [Setup your DNS A Record and Hostname](#Configure-DNS-A-Record)
   * Initial Email: This will be the **user email address** that will be deploying/setting up Custom Governance. Custom Governance will check for this email address even after the user has passed through IAP.
 
-To learn how to create these parameters go through the [installation process above](#marketplace-ui-deployment-details)
-
 #### Create a Google Kubernetes Engine cluster
 
 Create a cluster from the command line. If you already have a cluster that you
-want to use, this step is optional.
+want to use, this step is optional. You can also refer here for [instructions](#create-a-gke-cluster-with-custom-service-account) on how to create a GKE cluster with custom service account but ignore steps regarding the namespace and kubernetes service account because those will be covered in the following steps.
```

Comment:
somehow I can't comment on the command line below.
gcloud container clusters create "$CLUSTER" --scopes=https://www.googleapis.com/auth/cloud-platform --zone "$ZONE"
 is the bare minimum
I think it will be better to specify things like "service-account", machine type etc. And comment that those are optional.

Reply:
I think there are too many flags for the user to customize. I provide two examples and put the link https://cloud.google.com/sdk/gcloud/reference/container/clusters/create for them to refer.